database/migrations/0052-add_updated_at_to_contributions.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn(`ALTER TABLE \`contributions\` ADD COLUMN \`updated_at\` datetime DEFAULT NULL;`)
```

Comment:
`ADD COLUMN ... AFTER ...`

Reply:
`transactionId` like in the model? At least that seems the most intuitive to me.